aws_batch/pipeline.py

```diff
@@ -555,16 +554,16 @@ def run_sample(sample_s3_input_path, sample_s3_output_path,
                          "stats_file": os.path.join(result_dir, STATS_OUT)}
 
     # Download fastqs
-    command = "aws s3 ls %s/ |grep fastq.gz" % (sample_s3_input_path)
+    command = "aws s3 ls %s/ | grep %s" % (sample_s3_input_path, file_type)
```

Comment:
this is ok if filetype is one of (fasta, fastq.gz) but won't discern between fastq and fastq.gz

Reply:
also... ```fastanimation.fastq.gz```

may be just `grep '\.%s$'`

Reply:
Thanks!